pom.xml

```diff
@@ -180,16 +180,15 @@
         <objenesis.version>2.5</objenesis.version>
         <clojure.version>1.4.0</clojure.version>
         <clj-time.version>0.4.1</clj-time.version>
-        <commons-codec.version>1.10</commons-codec.version>
+        <commons-codec.version>1.11</commons-codec.version>
         <log4j.version>1.2.17</log4j.version>
         <commons-logging.version>1.2</commons-logging.version>
         <jsonSmart.version>2.3</jsonSmart.version>
         <minidev.accessors-smart.version>1.2</minidev.accessors-smart.version>
-        <ow2.asm.version>5.2</ow2.asm.version> <!-- version should align with cxf-rt-frontend-jaxws; see also json-path and karaf version requirements -->
+        <ow2.asm.version>5.2</ow2.asm.version> <!-- minidev.accessors-smart requires 5.x; aries-proxy:4.2.7 brings in 7.1 -->
         <commons-beanutils.version>1.9.3</commons-beanutils.version>
-        <javax.mail.version>1.4.4</javax.mail.version>
-        <cxf.javax.annotation-api.version>1.3</cxf.javax.annotation-api.version> <!-- cxf-specs feature v3.2.7 declares v1.3 (as does org.apache.cxf:cxf-rt-frontend-jaxrs), but karaf 4.2.2 has 1.2 -->
-        <plantuml.version>6121</plantuml.version>
+        <javax.mail.version>1.4.7</javax.mail.version> <!-- version should align with 'jetty' feature -->
+        <cxf.javax.annotation-api.version>1.3.1</cxf.javax.annotation-api.version> <!-- cxf-specs feature v3.3.2 declares v1.3.1; jetty 9.4.20.v20190813 declares v1.3 -->
```

Comment:
Note I wasn't sure about `cxf.javax.annotation-api.version` version 1.3.1, but I think it looks ok.

Here's what karaf shows:

```
bundle:list -t 0 -s | grep javax.annotation-api
 15 │ Active   │  80 │ 1.3.1                    │ javax.annotation-api
```

```
bundle:headers 15

javax.annotation API (15)
-------------------------
Archiver-Version = Plexus Archiver
Automatic-Module-Name = java.annotation
Bnd-LastModified = 1505305346644
Build-Jdk = 1.8.0_121
Built-By = sameerpandit
Created-By = 1.8.0_121 (Oracle Corporation)
Extension-Name = javax.annotation
Implementation-Vendor = GlassFish Community
Implementation-Vendor-Id = org.glassfish
Implementation-Version = 1.3.1
Manifest-Version = 1.0
Specification-Vendor = Oracle Corporation
Specification-Version = 1.3
Tool = Bnd-0.0.255

Bundle-Description = Java(TM) Common Annotations 1.3 API Design Specification
Bundle-DocURL = https://javaee.github.io/glassfish
Bundle-License = https://github.com/javaee/javax.annotation/blob/master/LICENSE
Bundle-ManifestVersion = 2
Bundle-Name = javax.annotation API
Bundle-SymbolicName = javax.annotation-api
Bundle-Vendor = GlassFish Community
Bundle-Version = 1.3.1

Export-Package = 
	javax.annotation;version=1.3.1,
	javax.annotation.security;version=1.3.1,
	javax.annotation.sql;version=1.3.1
Import-Package = 
	javax.annotation;version=1.3.1,
	javax.annotation.security;version=1.3.1,
	javax.annotation.sql;version=1.3.1
```

```
karaf@brooklyn()> package:exports | grep javax.annotation
javax.annotation.processing                                       │ 1.0.0                    │ 0   │ org.apache.felix.framework
javax.annotation.security                                         │ 1.3.1                    │ 15  │ javax.annotation-api
javax.annotation.sql                                              │ 1.3.1                    │ 15  │ javax.annotation-api
javax.annotation                                                  │ 1.0.0                    │ 0   │ org.apache.felix.framework
javax.annotation                                                  │ 1.3.1                    │ 15  │ javax.annotation-api
```